### DISCUSSION TITLE
Tesla APIs for Stormwatch status?

### DISCUSSION THREAD

**Discussion by rakgupta** [vloschiavo/powerwall2#82] [Q&A]
Hello - does anyone know if there is an API that shows the storm watch status? I am trying to set an alert (email/SMS) when storm watch is activated/deactivated but cannot find one in the API documentation. Thanks

**Comment by emanuelbalea** [comment]
Or for that matter any api to turn it on/off? In NZ stormwatch does not work despite being marked as an available feature. Their data source probably sucks.
